process_jartic_archives_fast.py

```
@@ -0,0 +1,217 @@
+        parts = archive.stem.split('_')
+        if len(parts) >= 4:
+            try:
+                year = int(parts[2])
+                month = int(parts[3])
+                archive_date = datetime(year, month, 1, tzinfo=timezone.utc)
+                
+                if (archive_date.year == args.start.year and archive_date.month >= args.start.month) or \
+                   (archive_date.year == args.end.year and archive_date.month <= args.end.month) or \
+                   (args.start.year < archive_date.year < args.end.year):
+                    filtered_archives.append(archive)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)